### PR TITLE
infra: Clone repo for cockpit lib update with token

### PR DESCRIPTION
**Pull request by VladimirSlavik**
This addresses https://github.com/rhinstaller/anaconda/pull/4790#issuecomment-1564673883

It's not clear if this solves the problem.

Currently blocked because we can't update.

### CODE REVIEW

File: .github/workflows/cockpit-lib-update.yml [hunk 1]
```diff
@@ -24,6 +25,8 @@ jobs:
 
       - name: Clone repository
         uses: actions/checkout@v3
+        with:
+          token: ${{ secrets.INSTALLKER_TOKEN }}
```
Comment (KKoukiou) [inline .github/workflows/cockpit-lib-update.yml:29]: I think you have a typo here? INSTALLKER?

Also the fact that we can't update is not making this PR blocked. We can nicely generate the PRs just not merge them.

**Review by KKoukiou — COMMENTED** [review]
All in all once this is tested I am ok that it get's merged even though we should not update for the time being.

File: .github/workflows/cockpit-lib-update.yml (hunk 1 above)
Comment (VladimirSlavik) [reply .github/workflows/cockpit-lib-update.yml:29]: That's the bot name, so its token has its name.

**Comment by VladimirSlavik** [comment]
Yup, not a high priority, just wanted to dump it here as a PR instead of a random comment hidden somewhere.

**Review by M4rtinK — APPROVED** [review]
Looks good to me. :)

**Comment by VladimirSlavik** [comment]
I think let's try and see what this does. Usually, we catch the issues with opening PRs later after we deploy the code.

The interaction with other tokens used throughout the whole process is unclear, so I don't see testing it as economical...

**Pull request merged**